hieraserver/rest/rest.go

```diff
@@ -49,6 +50,7 @@ func newCommand() *cobra.Command {
 	flags.StringVar(&config, `config`, `/hiera/hiera.yaml`, `path to the hiera config file. Overrides /hiera/hiera.yaml`)
 	flags.StringArrayVar(&cmdOpts.VarPaths, `vars`, nil, `path to a JSON or YAML file that contains key-value mappings to become variables for this lookup`)
 	flags.StringArrayVar(&cmdOpts.Variables, `var`, nil, `variable as a key:value or key=value where value is a literal expressed in Puppet DSL`)
+	flags.StringVar(&addr, `addr`, `127.0.0.1`, `ip address to listen on`)
```

Comment:
This will change the default from being an empty string to the ipv4 localhost. I'm not sure if that matters but I would prefer the empty string and let `http.ListenAndServe` make that decision.

Reply:
How about 0.0.0.0 default? That would match current behavior.

Reply:
Even so, why would 0.0.0.0 be better than the current empty string?

Reply:
No real reason, just a sense of beauty (default values displayed in --help).

Changed to the empty string.